src/administration/integrations.md

```diff
@@ -12,5 +12,4 @@ platform integrations
 ![Cli Integrations](/images/cli-integrations.png)
 
 > **note**
-> If you have created your account using the bitbucket or github oAuth Login in order to use the Platform CLI you will need to setup a
-> password which you can do by visiting this page [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password)
+> If you have created your account using the Bitbucket or GitHub oAuth Login, then in order to use the Platform CLI, you will need to set up a password by visiting [https://accounts.platform.sh/user/password](https://accounts.platform.sh/user/password).
```

Comment:
The comma after "CLI" is unnecessary.  The one after Login is technically optional in English, probably, but the latter is definitely unnecessary.  (It would be appropriate if there was also a comma after then, but that would be excessive.)